secret.config.sample.json

```diff
@@ -1,5 +1,5 @@
 {
-  "secret": "someSecretStringGoesHere",
-  "emailUsername": "someAccount@gmail.com",
-  "emailPassword": "somePassword"
+  "secret": "georgepburdell",
+  "emailUsername": "GaTechThriftShop@gmail.com",
+  "emailPassword": "8O1MoOiCkbPU4J2G3iR"
```

Comment:
whoops

Reply:
Gonna have to change the password and the secret

Reply:
Next time put this info in a file called "secret.config.json" not into "secret.config.sample.json"

Reply:
ok got it